docs/Training-ML-Agents.md

```diff
@@ -1,196 +1,116 @@
+  pressing the Play button in the Unity Editor"_ is displayed on the screen.
+* `<run-identifier>` is a unique name you can use to identify the results of your training runs.
+
+See the [Getting Started Guide](Getting-Started.md#training-a-new-model-with-reinforcement-learning)
+for a sample execution of the `mlagents-learn` command.
+
+#### Observing Training
+
+Regardless of which training methods, configurations or hyperparameters you provide,
+the training process will always generate three artifacts:
+1. Summaries (under the `summaries/` folder): these are training metrics that are updated
+throughout the training process. They are helpful to monitor your training performance
+and may help inform how to update your hyperparameter values.
+See [Using TensorBoard](Using-Tensorboard.md) for more details on how to visualize
+the training metrics.
+1. Models (under the `models/` folder): these contain the model checkpoints that are updated
+throughout training and the final model file (`.nn`). This final model file is generated once
+either when training completes or is interrupted.
+1. Timers file (also under the `summaries/` folder): this contains aggregated metrics on your
+training process, including time spent on specific code blocks.
+See [Profiling in Python](Profiling-Python.md) for more information on the timers generated.
+
+These artifacts (except the `.nn` file) are updated throughout the training process and finalized
+when training completes or is interrupted.
+
+#### Debugging
 
-**Note:** The default port TensorBoard uses is 6006. If there is an existing session
-running on port 6006 a new session can be launched on an open port using the --port
-option.
+If you enable the `--debug` flag in the command line, the trainer metrics are logged to a CSV file
+stored in the `summaries` directory. The metrics stored are:
+  * brain name
+  * time to update policy
+  * time since start of training
+  * time for last experience collection
+  * number of experiences used for training
+  * mean return
 
-When training is finished, you can find the saved model in the `models` folder
-under the assigned run-id — in the cats example, the path to the model would be
-`models/cob_1/CatsOnBicycles_cob_1.nn`.
+This option is not available currently for Behavioral Cloning.
 
-While this example used the default training hyperparameters, you can edit the
-[trainer_config.yaml file](#training-config-file) with a text editor to set
-different values.
+#### Stopping and Resuming Training
 
-To interrupt training and save the current progress, hit Ctrl+C once and wait for the
-model to be saved out.
+To interrupt training and save the current progress, hit `Ctrl+C` once and wait for the
+model(s) to be saved out.
 
-### Loading an Existing Model
+To resume a previously interrupted or completed training run, use the `--resume` flag and
+make sure to specify the previously used `run-id`.
```

Comment:
Should be consistent when referring to the run ID in the docs? I think we use a mix of `<run-identifier>`, `run-id`, "run-id", and "run ID".  I'm biased towards using `<run-identifier>` when talking about CLI commands and "run ID" or "run identifier" in paragraphs, but OK with the other options as well.

Reply:
Good point. I made a quick fix - let me know if this is what you had in mind.

Reply:
👍